src/AboutPage.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "styled-components";
+import Image from "react-bootstrap/Image";
+
+import PageLayout from "./PageLayout";
+
+const CreatorLayout = styled.div`
+  margin-top: 16px;
+  display: flex;
+  flex: column nowrap;
+  @media screen and (min-width: 840px) {
```

Comment:
Did you mean 480px here? Or is there a reason for the different size constraint

Reply:
That was a placeholder that I forgot to change. I think I'm going to put the change from grid to flex at a tablet size breakpoint since otherwise the columns get way too small. I'll change it to 768px which is the bootstrap value.